### ISSUE TITLE
“What’s new” button opens a new tab everytime it's pressed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#5548]
Logged via: https://bugzilla.mozilla.org/show_bug.cgi?id=1583599

User Agent: Mozilla/5.0 (iPad; CPU OS 12_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/20.0b16134 Mobile/15E148 Safari/605.1.15

Steps to reproduce:

    Tap on 3bar icon (hamburger?)
    Tap on new [What’s New] menu item
    Repeat steps 1. and 2.

Actual results:

A new what,s new tab appears every time

Expected results:

Opened it once and went to the original if address wasn’t modified.

**Comment by KrystynaKruchkovska** [comment]
I can reproduce the bug, I also notice that "what's new" link is redirected, so the problem is how to compare parent url with redirected one so as to "recognise" that 'Whats New' tab is already opened . Maybe anyone have some suggestions ?

**Comment by garvankeeley** [comment]
Agreed that because of the redirect we can't easily know that we opened this tab already.
